extensions/adapters/vector/src/main/java/org/locationtech/geowave/adapter/vector/query/aggregation/VectorMathAggregation.java

```diff
@@ -53,38 +70,24 @@ public void setParameters(final FieldNameParam fieldNameParam) {
   }
 
   @Override
-  public BigDecimal merge(final BigDecimal result1, final BigDecimal result2) {
-    return agg(result1, result2);
+  public BigDecimal getResult() {
+    return value;
   }
 
   @Override
-  public BigDecimal getResult() {
-    return value;
+  public BigDecimal merge(final BigDecimal result1, final BigDecimal result2) {
+    return agg(result1, result2);
   }
 
+  // TODO: is it OK that I changed these? I fixed the tests.
```

Comment:
I added VarintUtils.read/writeBigDecimal, so I changed it here too. Is that alright?

Reply:
Resolved through on of JD's comments